contentctl/objects/unit_test_result.py

```diff
@@ -37,18 +37,18 @@ def set_job_content(
         self.duration = round(duration, 2)
         self.exception = exception
         self.status = status
-
+        self.job_content = content
+        
         # Set the job content, if given
         if content is not None:
-            self.job_content = content
-
             if self.status == TestResultStatus.PASS:
                 self.message = "TEST PASSED"
             elif self.status == TestResultStatus.FAIL:
                 self.message = "TEST FAILED"
             elif self.status == TestResultStatus.ERROR:
-                self.message == "TEST FAILED (ERROR)"
+                self.message = "TEST FAILED (ERROR)"
```

Comment:
nit: I personally would just list this as `TEST ERROR` for conciseness

Reply:
Agreed, especially since that aligns with the TestResultStatus.ERROR.
I have made that change